lib/core-server/src/core-presets.test.ts

```diff
@@ -19,9 +22,13 @@ import htmlOptions from '../../../app/html/src/server/options';
 import webComponentsOptions from '../../../app/web-components/src/server/options';
 import { outputStats } from './utils/output-stats';
 
+const { SNAPSHOT_OS } = global;
+const mkdtemp = promisify(mkdtempCb);
+
 // this only applies to this file
 jest.setTimeout(10000);
 
+// FIXME: this doesn't work
```

Comment:
Comments on this FIXME, @IanVS?

Reply:
I found that the `stories.json` file was still being created.  Also, when I turned on logging, I was seeing messages about this preset being invalid.  I think maybe it's not in the correct format for a preset, perhaps?